taca/utils/bioinfo_tab.py

```diff
@@ -234,6 +256,42 @@ def get_status_element(aviti_run):
     return status
 
 
+def get_status_ont(ont_run):
+    """Gets status of a ONT sample run, based on flowcell info."""
+    # Default state, should never occur
+    status = "ERROR"
+    run_status = ont_run.get_demultiplexing_status()
+
+    if run_status in ["finished"]:
+        status = "New"
+    elif run_status in ["ongoing"]:
+        status = "Sequencing"
+
+    return status
+
+
+def get_ss_projects_ont(ont_run, couch_connection):
+    """Fetches project, FC, lane & sample (sample-run) status for a given folder for ONT runs"""
+    proj_tree = Tree()
+    flowcell_id = ont_run.run_name
+    flowcell_info = (
+        couch_connection["nanopore_runs"].view("info/lims")[flowcell_id].rows[0]
+    )
+    if flowcell_info.value and "sample_data" in flowcell_info.value["loading"][0]:
+        samples = flowcell_info.value["loading"][0]["sample_data"]
```

Comment:
I think the index should be `[-1]`, not `[0]`. If the LIMS script is re-run manually, we should use the latest values.